clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -799,12 +813,17 @@ boolean handleUnsupportedVersionException(UnsupportedVersionException exception)
 
         @Override
         public String toString() {
-            return "Call(callName=" + callName + ", deadlineMs=" + deadlineMs + ")";
+            return "Call(callName=" + callName + ", deadlineMs=" + deadlineMs +
+                ", tries=" + tries + ", nextAllowedTryMs=" + nextAllowedTryMs + ")";
         }
 
         public boolean isInternal() {
             return internal;
         }
+
+        public int tries() {
```

Comment:
nit: I think technically we don't need this. For internal classes, I tend to favor accessing fields directly when the class is nested. I don't feel strongly, so up to you.

Reply:
I do have a tendency towards setters and getters, but in this case since it is a nested class, we can remove it. Will remove it.

Reply:
Actually, since `tries` is a private field inside `Call`, I have the `tries()` getter method in order to pass the state of the number of tries across calls. In a future PR we could remove this when we have a better `CallContext` object as part of the exponential backoff KIP.